src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -334,9 +334,11 @@ def test_graph_group_scenario(self):
             return  # this test delete users which are beyond a SP's capacity, so quit...
 
         domain = username.split('@', 1)[1]
+        if domain == 'microsoft.com' and self.is_live:
+            domain = self.cmd('rest --method get --url https://graph.windows.net/myorganization/domains?api-version=1.6 --query "value[?isVerified==`true`].name|[0]"').get_output_in_json()
```

Comment:
Another approach would be to call command 

```
> az ad signed-in-user show
{
  ...
  "userPrincipalName": "rolelivetest@azuresdkteam.onmicrosoft.com",
```

I tried this before but reverted the change given I always use a `@AzureSDKTeam.onmicrosoft.com` account.

Reply:
This domain is also a verified domain but is not a valid email address:

```
    {
      "authenticationType": "Managed",
      "availabilityStatus": null,
      "forceDeleteState": null,
      "isAdminManaged": true,
      "isDefault": false,
      "isDefaultForCloudRedirections": false,
      "isInitial": false,
      "isRoot": true,
>>    "isVerified": true,
>>    "name": "azdevextest.com",
      "passwordNotificationWindowInDays": null,
      "passwordValidityPeriodInDays": null,
      "state": null,
      "supportedServices": []
    }
```

Reply:
Reverted as other role tests with `@microsoft.com` username may have the same issue and tests should be done with `@AzureSDKTeam.onmicrosoft.com` account for now.